elements/pom.xml

```diff
@@ -74,7 +74,7 @@
 		<dependency>
 			<groupId>org.jsoup</groupId>
 			<artifactId>jsoup</artifactId>
-			<version>1.8.1</version>
+			<version>1.11.2</version>
```

Comment:
Should maybe not define any JSoup version at all here, since any explicitly chosen version might be incompatible with what the used Vaadin 8 version uses. 

I don't think an explicit dependency is needed here since JSoup should come in as a transitive dependency from `vaadin-server`?

Reply:
yes, that's right, the dependency comes already with vaadin-server